build.gradle

```diff
@@ -1,163 +1,37 @@
 plugins {
     id 'java-library'
-    id 'signing'
-    id 'maven-publish'
-    id "io.github.gradle-nexus.publish-plugin" version "1.0.0"
-    id "org.owasp.dependencycheck" version "8.1.0"
+    id "com.exactpro.th2.gradle.publish" version "0.2.0"
 }
 
 group = 'com.exactpro.th2'
 version = release_version
 
-ext {
-    sharedDir                   = file("${project.rootDir}/shared")
-}
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
 
 repositories {
     mavenCentral()
-    configurations.all {
+    maven {
+        name 'Sonatype_snapshots'
+        url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+    }
+    maven {
+        name 'Sonatype_releases'
+        url 'https://s01.oss.sonatype.org/content/repositories/releases/'
+    }
```

Comment:
note: we don't really need those repositories here since we don't have any dependencies on our artifacts